package.json

```diff
@@ -14,41 +14,32 @@
     "@hoodie/account": "^2.0.0",
     "@hoodie/admin": "^1.0.1",
     "@hoodie/client": "^5.0.0",
+    "@hoodie/server": "^20.0.0",
     "@hoodie/store": "^1.0.1",
     "async": "^2.0.0",
-    "boom": "^3.1.2",
     "browserify": "^13.0.1",
     "good": "^6.6.3",
     "good-squeeze": "^4.0.0",
-    "h2o2": "^5.1.0",
     "hapi": "^13.3.0",
     "hapi-cors-headers": "^1.0.0",
     "inert": "^4.0.0",
-    "joi": "^9.0.0",
-    "jsonfile": "^2.3.0",
     "lodash": "^4.11.2",
-    "lout": "^9.0.0",
-    "memdown": "^1.1.2",
-    "mkdirp": "^0.5.1",
     "node-emoji": "^1.3.0",
     "npmlog": "^4.0.0",
-    "pouchdb": "^5.3.2",
-    "pouchdb-admins": "^1.0.1",
-    "pouchdb-users": "^1.0.3",
-    "randomstring": "^1.1.4",
+    "pouchdb-adapter-http": "^5.4.5",
+    "pouchdb-adapter-leveldb": "^5.4.5",
+    "pouchdb-adapter-memory": "^5.4.5",
+    "pouchdb-core": "^5.4.5",
```

Comment:
this feels so good. All hail @nolanlawson! Can’t wait to drop `leveldb` in favor of [pouchdb-adapter-fs](https://www.npmjs.com/package/pouchdb-adapter-fs) – no more native builds during install \o/